packages/react-native/ReactCommon/react/renderer/graphics/ValueUnit.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+namespace facebook::react {
+
+enum class UnitType {
+  UNDEFINED,
+  POINT,
+  PERCENT
```

Comment:
nit: other Fabric enums are pascal case
```suggestion
  Undefined,
  Point,
  Percent,
```